docs/src/tutorials/npm-browser-packages/building-your-package.md

```diff
@@ -0,0 +1,24 @@
+# Building your package
+
+We've written our code so now we need to package it all up.
+
+We are writing a package that should be used in the browser, so we run this in our terminal:
+
+```bash
+$ wasm-pack build --scope MYSCOPE
```

Comment:
Could the `--scope MYSCOPE` be omitted and mentioned below as an extra flag? It'd be pretty sweet if we simply recommended `wasm-pack build`

Reply:
the primary issue here is people usually cut and paste a tutorials instructions. if they do that they'll get an error that the package already exists.. this way we can be sure they are always successful. maybe we can edit the tutorial to ask folks to pick a unique name! i'll leave this out of the edit and we can do it once this lands.